src/toil/jobStores/abstractJobStore.py

```diff
@@ -400,7 +407,7 @@ def _readFromUrl(cls, url, writable):
 
         :param writable: a writable stream
 
-        :return int: returns the size of the file in bytes
+        :return int, bool: the size of the file in bytes, the user executable permission bit
```

Comment:
Maybe `from typing import Tuple` and call this `Tuple[int, bool]`? Or just make the docstring say `:return:` and put the type as ` def _readFromUrl(cls, url, writable) -> Tuple[int, bool]:`.

I've never quite understood what language we're using to express the argument and return types, and the way you write things with the typing module is at least [documented](https://docs.python.org/3/library/typing.html#typing.Tuple).